overlord/devicestate/devicestate_install_mode_test.go

```diff
@@ -243,3 +245,111 @@ func (s *deviceMgrInstallModeSuite) TestInstallModeNotClassic(c *C) {
 	installSystem := s.findInstallSystem()
 	c.Assert(installSystem, IsNil)
 }
+
+func (s *deviceMgrInstallModeSuite) TestInstallModeEncrypted(c *C) {
```

Comment:
This test and the following share a lot of setup code, I wonder if something could be extracted into shared code.

Reply:
With the change below there are new possibilities to be tested (secured and non-secured with forced encryption bypass or not) so I'll refactor the test code to handle it without too much redundancy.